401codechallenges/src/test/java/code401challenges/tree/BinaryTreeTest.java

```diff
@@ -18,23 +19,28 @@ public void setup () {
     }
 
     // Can return correct array for inOrder traversal
-    @Test public void traverseInOrder() {
+    @Test public void testTraverseInOrder() {
         assertEquals("The linked list should be 4 9 -219",
                 "[9, 4, -219]",
                 testTree.inOrder().toString());
     }
 
     // Can return correct array for preOrder traversal
-    @Test public void traversePreOrder() {
+    @Test public void testTraversePreOrder() {
         assertEquals("The linked list should be 4 9 -219",
                 "[4, 9, -219]",
                 testTree.preOrder().toString());
     }
 
     // Can return correct array for postOrder traversal
-    @Test public void traversePostOrder() {
+    @Test public void testTraversePostOrder() {
         assertEquals("The linked list should be 4 9 -219",
                 "[9, -219, 4]",
                 testTree.postOrder().toString());
     }
+
+    // Can return the correct print out of values for breadth first traversal
+    @Test public void testBreadthFirstTraversal(BinaryTree testTree) {

```

Comment:
this test will not work because traverse does not return a string, it returns null. If you would like, modify the breadthFirstTraversal to return a string representation of the traversal in addition to its other functionality.